source/nodejs/adaptivecards-site/themes/adaptivecards/source/js/script.js

```diff
@@ -475,10 +475,18 @@ $(function () {
 
 	});
 
+	$("button.copy-data").click(function (e) {
+		var content = $(this).parent().next("pre").text();
+		copyToClipboard(content);
+		$(this).append("<span role='status' aria-label='Sample data copied successfully' id='dataCopySuccess' />");
```

Comment:
>$(this).append("<span role='status' aria-label='Sample data copied successfully' id='dataCopySuccess' />"); [](start = 2, length = 107)

Does this result in a new span being added for each button click in such a way that they pile up?

Reply:
it does, but it's not really a concern -- there's no real end user impact... the message is read only when the element is added to the DOM.